src/components/City.jsx

```diff
@@ -0,0 +1,77 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import './City.css';
+import { v4 as uuidv4 } from 'uuid';
+import { fetchCityData } from '../redux/pollution/pollutionSlice';
+
+/* eslint-disable camelcase */
```

Comment:
I disabled the camelcase linter error because one of gas names form the API pm2_5 is not in in camelcase. I can't fix that, because, it's from the API side. That's why I disable it.

Reply:
@wacoo Please rename it to something more meaning by using if else condition. I believe that will work. Whatever you mentioned if it's true then disable the specific line not whole function. Try to resolve it first then go with the final solution